CorsixTH/Lua/languages/english.lua

```diff
@@ -911,7 +911,11 @@ map_editor_window = {
     heliport_3 = "Heliport 3",
     heliport_4 = "Heliport 4",
     paste = "Paste area",
-  }
+  },
+  checks = {
+    spawnpoints = "Warning: Spawn points could not be generated for this map.",
+    pathfind = "Warning: A path could not be made between the spawn points and the hospital.",
```

Comment:
"A path" or or "No path" ? Makes a big difference :)

I don't quite buy "made" either , it would work if you changed all outside tiles to road, right?

Perhaps "Warning: No path exists from  the edge of the map to the hospital" ?

or "no path available" could work too.